dotCMS/src/main/java/com/dotcms/graphql/datafetcher/StoryBlockFieldDataFetcher.java

```diff
@@ -16,7 +16,9 @@ public Map<String, Object> get(DataFetchingEnvironment environment) throws Excep
         final Map<String, Object> storyBlockMap = new HashMap<>();
 
         final String contFieldValue = contentlet.getStringProperty(variableName);
-        final JSONObject jsonContFieldValue = new JSONObject(contFieldValue);
+        final JSONObject jsonContFieldValue = contFieldValue!=null
+                ? new JSONObject(contFieldValue)
```

Comment:
Let's make `JSONObject` more robust if a null is sent in, have it return just an empty JSONObject

Reply:
@wezell Done. Validated at JSONObject level.